module/Photo/src/Photo/Form/PhotoImport.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Photo\Form;
+
+use Zend\Form\Form;
+use Zend\InputFilter\InputFilter;
+use Zend\Mvc\I18n\Translator as Translator;
+
+class PhotoImport extends Form
+{
+
+    public function __construct(Translator $translate)
+    {
+        parent::__construct();
+
+        $this->add(array(
+            'name' => 'album_id',
+            'type' => 'text',
+            'value' => '1' //TODO: get album id
+        ));
+
+        $this->add(array(
+            'name' => 'folder_path',
+            'type' => 'text',
+            'options' => array(
+                'label' => $translate->translate('Folder path')
+            )
+        ));
+
+        $this->add(array(
+            'name' => 'submit',
+            'type' => 'submit',
+            'options' => array(
+                'label' => $translate->translate('Import folder')
+            )
+        ));
+
+        $this->initFilters();
+    }
+
+    protected function initFilters()
+    {
+        $filter = new InputFilter();
+        //TODO: Validators!!
```

Comment:
Dit wilden we uitstellen totdat we daadwerkelijk de admin interface bouwen aangezien de form dan nog wel een beetje zal veranderen.